cypress/e2e/auth_spec/redirect.cy.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference types="cypress" />
+
+import { cy, describe, it, beforeEach } from "local-cypress";
+import LoginPage from "../../pageobject/Login/LoginPage";
+
+describe("redirect", () => {
+  const loginPage = new LoginPage();
+
+  beforeEach(() => {
+    cy.log("Logging in the user devdistrictadmin:Coronasafe@123");
```

Comment:
```suggestion
    cy.log("Logging in the user devdistrictadmin");
```

Let's not display the password in logs, serves no use.